pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -318,18 +319,50 @@ func (plugin *cniNetworkPlugin) TearDownPod(namespace string, name string, id ku
 	return plugin.deleteFromNetwork(plugin.getDefaultNetwork(), name, namespace, id, netnsPath, nil)
 }
 
-func podDesc(namespace, name string, id kubecontainer.ContainerID) string {
+func PodDesc(namespace, name string, id kubecontainer.ContainerID) string {
```

Comment:
@AlexeyPerevalov I still think this doesn't need to be capitalized/exported since all the users are in the `package cni`.

Reply:
@dcbw, I forgot to ask (pkg/kubelet/util/format/pod.go) has a similar function, podDesc. Maybe reuse it.